packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1524,6 +1526,9 @@ export const createAzureCommunicationCallAdapterFromClient: (
   options?
 ): Promise<CallAdapter> => {
   const deviceManager = (await callClient.getDeviceManager()) as StatefulDeviceManager;
+  deviceManager.getCameras();
+  deviceManager.getMicrophones();
```

Comment:
what's the reason we need to getmics and getspeakers for this change?

Reply:
without these fetches if we call `joinCall()` to skip the screen there will be no camera or mics in the adapter. we should be fetching the devices as soon as the `DeviceManager` is made